src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  addMovie: (newMovie: Movie) => void,
+};
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMessage, setErrorMessage] = useState(false);
```

Comment:
```suggestion
  const [isErrorMessage, setIsErrorMessage] = useState(false);
```
it's a boolean value, so better to name it it this way